lib/impress.application.js

```diff
@@ -110,38 +96,35 @@ impress.application.mixin = (application) => {
   application.relative = (
     path // absolute path
     // Return: application relative path
-  ) => path.substr(application.dir.length);
+  ) => (
+    path.substr(application.dir.length)
+  );
 
   application.prepareScript = (
     fileName, // file name (absolute path)
     source // JavaScript code
     // Return: exported function or object exported from script
   ) => {
     const fn = application.relative(fileName);
-    const scriptDir = (
-      application.isImpress ? impress.dir : impress.applicationsDir
-    );
-    const scriptName = fileName.replace(scriptDir, '');
     try {
-      let code;
-      if (source[0] === impress.ASCII_BRACE) {
-        code = impress.USE_STRICT + '(' + source + ')';
-      } else {
-        code = impress.USE_STRICT + source;
-      }
-      const script = api.vm.createScript(code, scriptName);
+      if (source[0] === impress.ASCII_BRACE) source = '(' + source + ')';
```

Comment:
Which brace? I think it is better to rename the constant to `ASCII_OPENING_BRACE` or `ASCII_BRACE_OPENING`.